docs/content/simulation.tex

```diff
@@ -0,0 +1,164 @@
+\section{Simulation}
+
```

Comment:
As an introduction, explain quickly what we are trying to achieve with this "simulation" thing